modules/querier/config.go

```diff
@@ -17,6 +17,7 @@ type Config struct {
 	ExtraQueryDelay         time.Duration `yaml:"extra_query_delay,omitempty"`
 	MaxConcurrentQueries    int           `yaml:"max_concurrent_queries"`
 	Worker                  worker.Config `yaml:"frontend_worker"`
+	QuerySingleIngester     bool          `yaml:"query_single_ingester"`
```

Comment:
Let's change this parameter name. When using RF3 your change would actually be querying 3 ingesters, not 1.

I'm not coming up with a good name. If you have a good idea, please use it over :).
`query_ingesters_from_ring`? `query_relevant_ingesters`?